docs/telemetry.md

```diff
@@ -0,0 +1,44 @@
+# Telemetry
+
+The Telemetry function enables the Operator gathering and sending basic anonymous data to Percona, which helps us to determine where to focus the development and what is the uptake for each release of Operator.
+
+The following information is gathered:
+
+* ID of the Custom Resource (the `metadata.uid` field)
+* Kubernetes version
+* Platform (is it Kubernetes or Openshift)
+* PMM Version
+* Operator version
+* Percona XtraDB Cluster version
+* HAProxy version
+* ProxySQL version
+* Percona XtraBackup version
```

Comment:
also if cluster wide deployment used or not?

Reply:
added